### PR TITLE
Fixed 500 error in MultiTableQuery page

### LINKED ISSUE #14298: Query > Submit query : Error 500
### Steps to reproduce 1
1. https://demo.phpmyadmin.net/master-config/db_multi_table_query.php?db=world
2. Submit query
### Steps to reproduce 1bis
1. https://demo.phpmyadmin.net/master-config/db_multi_table_query.php?db=world
2. Select table
3. Submit query

### Expected behaviour
Case 1: Show a warning or do like https://demo.phpmyadmin.net/QA_4_8/db_multi_table_query.php?db=world
Case 1bis: Update and submit the query

### Actual behaviour
Error 500

### Server configuration
**phpMyAdmin version:** 5.0.0

### PR DESCRIPTION
**Pull request by Gaurav-Punjabi**
Fixed the 500 error when a empty query is passed to MultiTableQuery.
This is in reference to solve issue #14298.

### Description
Fixed error 500 on MultiTableQuery page when a empty query is passed.

Please describe your pull request.

Fixes #14298 

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: js/db_multi_table_query.js [hunk 1]
```diff
@@ -118,6 +118,12 @@ AJAX.registerOnload('db_multi_table_query.js', function () {
 
     $('#submit_query').on('click', function () {
         var query = editor.getDoc().getValue();
+        // Verifying that the given query is not empty
+        if (query === '') {
+            PMA_ajaxShowMessage('Please enter the sql query first.', false, 'error');
```
Comment (williamdes) [inline js/db_multi_table_query.js]: Can you use `PMA_messages` like in this example (for translation) https://github.com/phpmyadmin/phpmyadmin/blob/4d8310432f2c46a425cd1a61c5fec3b9e82b82a7/js/db_search.js#L55

Thank you for the PR :)

**Review by williamdes — CHANGES_REQUESTED** [review]

File: js/db_multi_table_query.js (hunk 1 above)
Comment (Gaurav-Punjabi) [reply js/db_multi_table_query.js]: Yeah sure

**Review by williamdes — APPROVED** [review]
LGTM
Thank you !

**Review by ibennetch — CHANGES_REQUESTED** [review]

File: js/messages.php [hunk 2]
```diff
@@ -139,6 +139,9 @@ function () {
 /* l10n: Default label for the y-Axis of Charts */
 $js_messages['strYValues'] = __('Y values');
 
+/* DB MULTI TABLE QUERY */
```
[the remark this answers is not in the document]
Comment (Gaurav-Punjabi) [reply js/messages.php]: Okay no problem

File: js/messages.php [hunk 3]
```diff
@@ -139,6 +139,9 @@ function () {
...
+$js_messages['strEmptyQuery'] = __('Please enter the sql query first.');
```
[the remark this answers is not in the document]
Comment (Gaurav-Punjabi) [reply js/messages.php]: Yeah sure. Will change it

[the remark this answers is not in the document]
Comment (Gaurav-Punjabi) [reply js/messages.php]: Done made the changes

**Review by williamdes — APPROVED** [review]

**Review by MauricioFauth — APPROVED** [review]

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**